source/administration/bulk-export-data.rst

```diff
@@ -200,11 +200,88 @@ User object
       <td valign="middle">array</td>
       <td>The teams which the user is member of. Is an array of <b>UserTeamMembership</b> objects.</td>
     </tr>
+    <tr class="row-odd">
+      <td valign="middle">notify_props</td>
+      <td valign="middle"><b>UserNotifyProps</b> object</td>
+      <td>The user’s notify props, as defined by the <b>UserNotifyProps</b> object.</td>
```

Comment:
`The user’s notify props` --> 0/5, wondering if this is clear to admins on first read what it refers to?

Reply:
Thanks @jasonblais updated based on an entry in the bulk import doc.  Also updated the bulk import document where it was similar to this entry so they are all consistent.